frontend/ui/uimanager.lua

```diff
@@ -467,16 +467,16 @@ function UIManager:close(widget, refreshtype, refreshregion, refreshdither)
     end
     logger.dbg("close widget:", widget.name or widget.id or tostring(widget))
     local dirty = false
-    -- Ensure all the widgets can get onFlushSettings event.
+    -- First notify the closed widget to save its settings...
     widget:handleEvent(Event:new("FlushSettings"))
-    -- first send close event to widget
+    -- ...and notify it that it ought to be gone now.
     widget:handleEvent(Event:new("CloseWidget"))
-    -- make it disabled by default and check if any widget wants it disabled or enabled
+    -- Make it disabled by default and check if any widget wants it disabled or enabled
```

Comment:
If you're going to change the line, change the line. :-P
```suggestion
    -- Make it's disabled by default and check if there are any widgets that want it disabled or enabled.
```